examples/existing-kms/README.md

```diff
@@ -0,0 +1,6 @@
+# Example that uses existing KMS instance
+
+An example that uses the module's default variable values.
```

Comment:
It does not use only default variable value but KMS guid needs to be provided by user. So need to change context accordingly.